pygeo/parameterization/DVGeoCST.py

```diff
@@ -1153,7 +1192,7 @@ def computeCSTdydN2(x, N1, N2, w, dtype=float):
         return dydN2
 
     @staticmethod
-    def computeCSTfromCoords(xCoord, yCoord, nCST, N1=0.5, N2=1.0, dtype=float):
+    def computeCSTfromCoords(xCoord, yCoord, yTE, nCST, N1=0.5, N2=1.0, dtype=float):
```

Comment:
What is `yTE`? If it's the trailing edge thickness, isn't that something that's possible to detect directly from the y coordinate at the trailing edge? That way we could maintain the same arguments as before and it'll automatically subtract out the appropriate value.

Reply:
Another thought if you want to maintain the ability to specify `yTE`: Make it a keyword argument that defaults to `None`, in which case it uses the automatic method. If the user specifies it, the specified value is used instead.

Reply:
Good point, no need for the additional argument